src/test/kotlin/com/openlattice/TestServer.kt

```diff
@@ -66,7 +65,6 @@ open class TestServer {
             hds = testServer.context.getBean(HikariDataSource::class.java)
 
             testServer.context.getBean(EventBus::class.java)
-                    .register(PostgresEdmManager(hds, hazelcastInstance))
```

Comment:
🤔

Reply:
not sure why this was ever happening tbh?